doc/user/content/cloud/account-limits.md

```diff
@@ -9,21 +9,37 @@ menu:
 
 {{< cloud-notice >}}
 
+Materialize Cloud offers two deployment sizes for free trials, with additional processing and RAM available for a premium upgrade. We use AWS as the cloud service provider and the us-east-1 datacenter.
+
+Trials last for 30 days or until you max out the available resources for free deployments, whichever comes first. Accounts and deployments will be deleted 14 days after your trial expires unless you have upgraded to a premium version.
```

Comment:
@msudberg 

Per a sync with Nikhil yesterday, for the initial Open Beta release we are going to manually do a sweep of users whose accounts' free trials have expired, track how many days it has been since the trial expired, and engage them either via email or Slack to try and bring them back into the fold, ex. take a call to action step. Which is to say that we are not going to be implementing any automation to deactivate accounts and delete associated deployments at exactly 14 days. I don't think it's the worst thing to keep this 14 days marker but wanted to bring that to your attention all the same - thoughts?

Reply:
This is the work that will be done to enable the manual sweeps referenced in the above comment: [Enable for (internal) Admins list of users whose free trial has expired and for how long (how many days since expiration) #1058](https://github.com/MaterializeInc/cloud/issues/1058)